tools/openms/MSGFPlusAdapter.xml

```diff
@@ -2738,7 +2738,7 @@
         <option value="trifluoro (L)">trifluoro (L)</option>
       </param>
     </repeat>
-    <repeat name="rep_param_variable_modifications" min="0" max="1" title="param_variable_modifications">
```

Comment:
This also seems to be a bug in the CTD to Galaxy converter. Why should we using a repeat if there is only one allowed case.

Thanks @Stortebecker!

Reply:
It's the other way round: Many are allowed, no need for max, this should fix it: https://github.com/WorkflowConversion/CTDConverter/pull/34